pkg/relayer/proxy/proxy_test.go

```diff
@@ -462,6 +463,49 @@ func TestRelayerProxy_Relays(t *testing.T) {
 			expectedErrCode: 0,
 			expectedErrMsg:  "",
 		},
+		{
+			desc: "Successful late relay from session under grace period",
+
+			relayerProxyBehavior: []func(*testproxy.TestBehavior){
+				// blockHeight is past the first session but within its grace period
+				testproxy.WithRelayerProxyDependenciesAndCurrentBlockHeight(
+					supplierKeyName,
+					blockHeight+sessionkeeper.SessionGracePeriod,
+				),
+				testproxy.WithRelayerProxiedServices(proxiedServices),
+				testproxy.WithDefaultSupplier(supplierKeyName, supplierEndpoints),
+				testproxy.WithDefaultApplication(appPrivateKey),
+				// Add 2 sessions with the first one being past but within the grace period
+				// and the second one being the current session
+				testproxy.WithSuccessiveSessions(supplierKeyName, "service1", appPrivateKey, 2),
+			},
+			inputScenario: sendRequestWithCustomSessionHeight(blockHeight),
+
+			expectedErrCode: 0,
+			expectedErrMsg:  "",
+		},
+		{
+			desc: "Failed late relay from closed session",
+
+			relayerProxyBehavior: []func(*testproxy.TestBehavior){
+				// blockHeight is past the first session's grace period
+				testproxy.WithRelayerProxyDependenciesAndCurrentBlockHeight(
+					supplierKeyName,
+					// Set the current block height value returned by the block provider
+					blockHeight+sessionkeeper.NumBlocksPerSession+sessionkeeper.SessionGracePeriod,
```

Comment:
Can you move this "math" into a separate line and assign to a local variable please?